mkosi/__init__.py

```diff
@@ -1489,7 +1489,9 @@ def mount(
         run(cmd)
         yield where
     finally:
+        timestamp = os.path.getmtime(where)
         run(["umount", "--no-mtab", "--recursive", where])
+        os.utime(where, (timestamp, timestamp))
```

Comment:
what's the rationale here?

Reply:
This is because of the find hack previously.

We mount `tmpfs` mounts on a few mountpoints in the Arch installation, and the mtime/atime modifications done previously does doesn't propegate.

Reply:
But I think if reproducible stuff is requested we probably should create/update the mount point inodes with the right mtime already *before* mounting stuff over.

Reply:
I can try see if we can fit it in a better location while writing this into python.